File: live-test/go/negative_super.go
package main

// Standalone struct method with a uniquely-named method — no interface in
// this file declares Compute, so no super (Go uses implicit interfaces).
// Ground truth: the correct result is an empty super list (method resolves,
// zero supers), not an error — matching GoLand's absent gutter icon here.
type Standalone struct{}

func (s *Standalone) Compute() string {
	return "standalone"
}

File: live-test/rust/src/negative_super.rs
// Inherent impl (no trait) — fn has no super in any trait.
// Ground truth: the super-method result for foo() is EMPTY — correct, and it
// matches the gutter, which shows no marker on an inherent method. RustRover's
// raw go-to-super action instead jumps to the enclosing module declaration;
// that is module noise the provider filters out, not a missing super.
pub struct Inherent;

impl Inherent {
    pub fn foo(&self) -> String {
        "inherent".to_string()
    }
}
